### ISSUE TITLE
Update toolkit to include new branding

### ISSUE THREAD

**Issue by adamjohnson** [wvuweb/cleanslate-toolkit#9]
New branding found on https://brand.wvu.edu. Things to do:

  * [x] Integrate new typography styles
  * [x] Integrate WVU Masthead Mini (as seen on https://patterns.wvu.edu/patterns)
  * [x] Integrate new footer

All this is going to involve making a new [global stylesheet](https://stash.development.wvu.edu/projects/CST/repos/code/browse/stylesheets/global-stylesheet), likely v6.0.

We've been asked, "What's the future of the CleanSlate Toolkit now that [Brand Patterns](https://patterns.wvu.edu) exists?" The CleanSlate toolkit still represents a solid foundation for making a CleanSlate theme. Brand Patterns has a lot of moving parts and is a lot like Twitter Bootstrap—it's a framework to build off of. The CleanSlate toolkit is more of a boilerplate, a bare-bones toolkit to eliminate repetitive tasks at the beginning of projects. It's much less complex and also serves as a great starting point when teaching people how to make CleanSlate themes.

In summary, Brand Patterns is to Twitter Bootstrap as the CleanSlate Toolkit is the the HTML5 Boilerplate.

**Comment by adamjohnson** [comment]
fda09a86a9e471672f2aa733cdd13c06b9b5ae75 adds v6.0 of the global stylesheet and closes this issue.

**Comment by adamjohnson** [comment]
Blog post on this subject:

https://cleanslatecms.wvu.edu/blog/2015/06/16/global-stylesheet-v6
